playbooks/ReadMe.txt

```diff
@@ -0,0 +1,9 @@
+Automate provisioning OpenStack Instance
```

Comment:
nit: the file should be called `README.md`.  If you don't have experience formatting markdown, check out [this guide](https://guides.github.com/features/mastering-markdown/)